support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.tsx

```diff
@@ -246,34 +212,44 @@ function setSubsCardStartDateInState(
 // ----- Component ----- //
 function PaperCheckoutForm(props: PropTypes) {
 	useCsrCustomerData(props.setCsrCustomerData);
+
 	const days = getDays(props.fulfilmentOption, props.productOption);
 	const isHomeDelivery = props.fulfilmentOption === HomeDelivery;
+
 	const fulfilmentOptionDescriptor = isHomeDelivery
 		? 'Newspaper'
 		: 'Subscription card';
+
 	const deliveryTitle = isHomeDelivery
 		? 'Where should we deliver your newspaper?'
 		: 'Where should we deliver your subscription card?';
+
 	const submissionErrorHeading =
 		props.submissionError === 'personal_details_incorrect'
 			? 'Sorry there was a problem'
 			: 'Sorry we could not process your payment';
+
 	const paymentMethods = supportedPaymentMethods(
 		props.currencyId,
 		props.country,
 	);
+
 	const isSubscriptionCard = props.fulfilmentOption === Collection;
 	let formattedStartDate = '';
 
 	if (isSubscriptionCard) {
 		const timeNow = Date.now();
-		const startDate = getPaymentStartDate(timeNow, props.productOption);
+		const startDate = getPaymentStartDate(
+			timeNow,
+			props.productOption as ActivePaperProducts,
```

Comment:
again just want to check this cast is safe?

Reply:
what do you mean by safe?

Reply:
(see above)